SwiftEvolve/Sources/SwiftEvolve/Planner.swift

```diff
@@ -110,16 +111,17 @@ public class Planner<G: RandomNumberGenerator>: SyntaxVisitor {
     }
   }
   
-  public override func visitPre(_ node: Syntax) {
-    guard error == nil else { return }
+  public func visitAny(_ node: Syntax) -> SyntaxVisitorContinueKind {
+    guard error == nil else { return .visitChildren }
```

Comment:
This should probably be `skipChildren`; once we see an error, we just want to finish as quickly as possible.

Reply:
Done.